requirements.txt

```diff
@@ -8,4 +8,5 @@ transformers==4.19.2
 nbconvert==6.5.0
 jupyterlite==0.1.0b7
 lightning
+lightning-app
```

Comment:
isn't it installed with `lightning`?

Reply:
Oh, this was for testing. Sorry, I'll close this PR. @aniketmaurya